### DISCUSSION TITLE
Guidelines for PR Commits and Commit Messages

### DISCUSSION THREAD

**Discussion by liamfallon** [nephio-project/nephio#413] [Development]
ONAP had this page to help users write good commit messages. I think most of it is applicable to Nephio as well:
[ONAP Commit Message Help Page](https://wiki.onap.org/display/DW/Commit+Messages)

**Comment by electrocucaracha** [comment]
Adding more references to this topic:
* [Google Engineering Best practices](https://github.com/google/eng-practices/blob/master/review/developer/index.md)
* [OpenStack Commit Good Practice](https://wiki.openstack.org/wiki/GitCommitMessages)
